Gn feature/helper module

gabrielcnunez opened the pull request:
Added game_statistics helper module, spec file.  Also refactored count_games method.
Testing is showing 99.14% coverage for all methods.

gabrielcnunez committed:
Create game stat helper class, spec file

gabrielcnunez committed:
Update spec helper to include game spec

bflanagan138 reviewed (approved):
Changes look appropriate and according to group DTR conventions. Changes made as a group via zoom 9/19/22

josephhilby committed:
Update spec_helper.rb

Added `require_relative 'league_statistics_spec'`

josephhilby committed:
Update stat_tracker.rb

Added   `require_relative 'league_statistics'` `require_relative 'season_statistics'` `include LeagueStatistics` `include SeasonStatistics`

josephhilby reviewed (approved):
Additions as needed to account for previous PRs

bflanagan138 committed:
Merge branch 'main' into GN_feature/helper_module
